MMM-Wallpaper.js

```diff
@@ -143,6 +143,13 @@ Module.register("MMM-Wallpaper", {
       element.style.opacity = 1;
       self.title.style.display = "none";
 
+      if(self.config.crossfade){
+        images = document.getElementsByClassName("crossfade-image")
+        if(images.length > 1 ){
+          images[0].style.opacity = 0
+        }
+      }
```

Comment:
Should call this on `self.wrapper`, as there may be multiple MMM-Wallpaper modules on the page, and one shouldn't affect the others.  Also, should keep the formatting consistent with the rest of the file:

```suggestion
      if (self.config.crossfade) {
        images = self.wrapper.getElementsByClassName("crossfade-image");
        if (images.length > 1) {
          images[0].style.opacity = 0;
        }
      }
```